source/db-reference-validation.txt

```diff
@@ -91,8 +91,11 @@ The sample code attempts to call a ``restaurant_name`` collection:
 Because the collection doesn't exist in the ``sample_mflix`` database, the 
 {+intellij-short+} raises a warning that the collection cannot be resolved:
 
-``Cannot resolve "restaurant_name" collection in "sample_mflix" database in the 
-connected data source.``
+.. code-block:: none
```

Comment:
changed this format because in some screen sizes, the monospace text was getting cut off.